docs/Acceptable-Formulae.md

```diff
@@ -73,11 +73,11 @@ Clang is the default C/C++ compiler on macOS (and has been for a long time). Sof
 
 We're a package manager so we want to do things like resolve dependencies and set up applications for our users. If things require too much manual intervention then they aren't useful in a package manager.
 
-### Static libraries
+### Shared vs. static libraries
 
-In general, formulae should not ship static libraries since these cannot be updated without a rebuild of the dependant software.
-If a formula gets a lot of requests to install static libraries, they may be installed by the formula.
-Applications in homebrew/core linking against libraries should link against shared libraries not static versions.
+In general, if formulae have to ship either shared or static libraries: they should ship shared ones.
+If they can ship both, they should.
```

Comment:
That's introducing a bunch of extra code and maintenance. Let's just allow it and not mandate it

Reply:
@SMillerDev Can you suggest a rewording?
```suggestion
If they can ship both, they can.
```

?

Reply:
I'd say keeping the current situation is fine
```suggestion
If there is demand for a static library they can ship both.
```

Reply:
@SMillerDev I think whenever we talk about "demand" we tend to imply "by more than one person" which I don't think is/should be the case here. I get the desire to say that it's not a requirement but I also would rather we softened the language to make it clear we welcome these PRs.

Reply:
How about `s/demand/interest`

Reply:
Yup, that works.